jekyll/_ccie/overview.md

```diff
@@ -50,12 +69,17 @@ Our Builder Boxes handle running your builds, and store no state themselves. Eac
 | Services Box                     | all traffic / all ports | Internal Communication                                         |
 | Builder Boxes (including itself) | all traffic / all ports | Internal Communication                                         |
 
-#### GitHub
----
+CircleCI Enterprise uses GitHub Enterprise credentials for
+authentication which, in turn, may use LDAP, SAML, or SSH for access. The following table describes the ports used on machines running GitHub to communicate with the Services and Builder instances.
```

Comment:
added: That is, CircleCI will inherit the authentication supported by your central SSO infrastructure.  (I think this is ok because the section is about GitHub)